airbyte-cdk/java/airbyte-cdk/db-sources/src/testFixtures/java/io/airbyte/cdk/integrations/standardtest/source/AbstractSourceDatabaseTypeTest.java

```diff
@@ -224,16 +254,23 @@ protected String getValueFromJsonNode(final JsonNode jsonNode) throws IOExceptio
    * @throws Exception might raise exception if configuration goes wrong or tables creation/insert
    *         scripts failed.
    */
-  private void setupDatabaseInternal() throws Exception {
-    final Database database = setupDatabase();
-
-    initTests();
 
+  protected void createTables() throws Exception {
     for (final TestDataHolder test : testDataHolders) {
       database.query(ctx -> {
         ctx.fetch(test.getCreateSqlQuery());
-        LOGGER.debug("Table " + test.getNameWithTestPrefix() + " is created.");
+        LOGGER.info("Table {} is created.", test.getNameWithTestPrefix());
```

Comment:
Suppose the table creation actually fails, will this log message be emitted anyway? Probably not, but worth double checking, in case jooq is lazy (dunno if it is)